src/coreclr/vm/jitinterface.cpp

```diff
@@ -1950,7 +1950,7 @@ unsigned CEEInfo::getClassAlignmentRequirement(CORINFO_CLASS_HANDLE type, bool f
         result = getClassAlignmentRequirementStatic(clsHnd);
```

Comment:
The contract on `getClassAlignmentRequirementStatic` is wrong and it should be fixed as well. It should be `STANDARD_VM_CONTRACT;` or something else that throws.

Reply:
Do I just change getClassAlignmentRequirement to STANDARD_VM_CONTRACT as well?

Reply:
I would leave the longer version of the contract on `getClassAlignmentRequirement` for consistency with other methods in this file that implement JIT/EE interface. All of them seem to have the longer version of the contract. I am not sure whether there is a reason - I do not see any obvious one.

If we wanted to use shorter version of the contract on JIT/EE interface implementation methods, we may want to find&replace it on all of them in separate PR.